server/play/game.go

```diff
@@ -52,14 +52,20 @@ func CreateGame(players []model.Player, pAPIs map[model.PlayerID]interaction.Pla
 		PeggedCards:     make([]model.PeggedCard, 0, 4*len(players)),
 	}
 
-	err := runStartHandlers(&g, pAPIs)
-	if err != nil {
-		return model.Game{}, err
-	}
+	// TODO should we actually run start handlers upon game creation? This can lead to
```

Comment:
Can we leave this as it was written and 🤞 that transactional DB will do the trick?

Reply:
hrmmmm I already had to change it to make things work. (see `createGame` in `lambdas.go`)

Reply:
nvm, the goroutine action handling fixed the original need for this 😅

Reply:
FYI https://github.com/joshprzybyszewski/cribbage/pull/31